### ISSUE TITLE
RuntimeError: Error in dlopen or dlsym: libcaffe2_nvrtc.so: cannot open shared object file: No such file or directory (checkDL at /opt/conda/conda-bld/pytorch_1573049304260/work/aten/src/ATen/DynamicLibrary.cpp:21)

### ISSUE THREAD

**Issue by yonglezhang95** [zengxianyu/crfill#26]
When I run sh train.sh, I get the error: RuntimeError: Error in dlopen or dlsym: libcaffe2_nvrtc.so: cannot open shared object file: No such file or directory (checkDL at /opt/conda/conda-bld/pytorch_1573049304260/work/aten/src/ATen/DynamicLibrary.cpp:21).
But I can not solve this error.

**Comment by zengxianyu** [comment]
My guess is that you have some missing dependencies

**Comment by yonglezhang95** [comment]
> My guess is that you have some missing dependencies

yes, when I changed the version of Pytorch, the error is solved. However, I got another error "KeyError: 'conv1.weight'", when I ran "sh train.sh".
